_guides/analytics/deep_dive_analytics.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: page
+title: Deep Dive into AMP Analytics
+order: 1
+---
+
+This guide dives deep into the
+[amp-analytics component](/docs/reference/extended/amp-analytics.html),
+breaking up a sample `amp-analytics` configuration into these key building blocks:
+
+{% include toc.html %}
+
+The remainder of this guide uses this configuration sample,
+which tracks the page views and user clicks on links
+and sends the analytics data to the third-party provider,
+[Google Analytics](https://developers.google.com/analytics/devguides/collection/amp-analytics/):
+
+{% highlight html linenos %}
+<amp-analytics type="googleanalytics" config="https://example.com/analytics.account.config.json">
+<script type="application/json">
+{
+  "requests": {
+    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
+    "event": "https://example.com/analytics?eid=${eventId}&elab=${eventLabel}&acct=${account}"
+  },
+  "vars": {
+    "account": "ABC123"
+  },
+  "triggers": {
```

Comment:
I drafted this as a subsection in the requests section, and it seems a better fit in terms of relevancy.